tests/Feature/MyTests/NoteTest.php

```diff
@@ -0,0 +1,159 @@
+<?php
+
+namespace Tests\Feature;
+
+// use Illuminate\Foundation\Testing\RefreshDatabase;
+use App\Models\Note;
+use App\Models\User;
+use Egulias\EmailValidator\Parser\Comment;
+use Egulias\EmailValidator\Parser\CommentStrategy\LocalComment;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use PHPUnit\Framework\SkippedTest;
+use Tests\TestCase;
+
+/*
+    Tests will follow the AAA (Arange Act Assert) format
+*/
+
+class NoteTest extends TestCase
+{
+    use RefreshDatabase; //clears the database for each test run
+    protected $userOne, $userTwo;
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+        // Create 2 test users, need 2 so we can test note viewing authentication
+        $userData = [];
+
+        for ($i = 1; $i <= 2; $i++) {
+            $userData[] = [
+                'name' => 'Test User #' . $i,
+                'email' => 'testemail' . $i . '@test.com',
+                'password' => bcrypt('pass123.'),
+            ];
+        }
+
+        User::factory()->createMany($userData);
+        $this->userOne = User::find(1); //assign userOne
+        $this->userTwo = User::find(2);//assign userTwo
+
+        //Prefab'd notes for user using different looping method than user prefab loop
+        Note::factory(10)->create()->each(function ($note) {
+            $note->update([
+                'note' => 'this is note #' . $note->id . ' for user #' . $note->user_id,
+            ]);
+        });
+
+    }
+
+    public function test_application_response_if_user_isnt_logged_in()
+    {
+
+        $response = $this->get('/');
+        $response->assertStatus(302);
+
+        $response = $this->get('/login');
+        $response->assertStatus(200);
+    }
+
+    public function test_application_response_if_user_is_logged_in()
+    {
+        $this->actingAs($this->userOne);
+        $response = $this->get('/note');
+        $response->assertStatus(200);
+        $response->assertSee('Test User #1');
+    }
+
+    /*
+    Tests TODOS:
+        ✔ note view when logged in
+        ✔ note view when logged out/different user
+        ✔ note creation
+        ✔ note update
+        ✔ note delete
+    */
+
+    public function test_note_viewability_and_validity()
+    {
+        $this->actingAs($this->userOne);
+        $response = $this->get('/note/10');
+        $response->assertStatus(200);
+        $response->assertSee('this is note #10 for user #1'); //Correct note test
+        $response->assertSee('Edit'); //edit button present test
+        $response->assertSee('Delete'); //delete button present test
+    }
+
+    public function test_note_viewability_from_unathorized_user()
+    {
+        $this->actingAs($this->userTwo);
```

Comment:
Why is this user not authorized?

Reply:
Here I believe I named the test method/function incorrectly. This test is meant to show that a user cannot access a different user's note, not that any of the users are actually unauthorized. A better name for the method could be
`test_user_cannot_access_other_users_note()`